misk/src/main/kotlin/misk/MiskServiceModule.kt

```diff
@@ -29,6 +32,8 @@ class MiskRealServiceModule : KAbstractModule() {
     install(TickerModule())
     install(TokenGeneratorModule())
     install(MiskCommonServiceModule())
+    bind<MiskConcurrencyLimiterEnabledFeature>()
+      .to(RealMiskConcurrencyLimiterEnabledFeature::class.java)
```

Comment:
Should this be here? or should it go into Skim? If it were in skim, we could remove the dependency on misk-feature in the main misk module

Reply:
I like the idea to push it to `Skim` to remove the dependency here